CoreAlgorithmPlugins/src/au/gov/asd/tac/constellation/plugins/algorithms/sna/centrality/PagerankCentralityPlugin.java

```diff
@@ -44,7 +40,7 @@
  * Calculates pagerank centrality for each vertex. This centrality measure does
  * not include loops.
  *
- * @author cygnus_x-1
```

Comment:
You should add an additional author rather than replace the author (although i am happy for all bug reports to go to you :p).

Reply:
Oh would you look at that, your on the author list again 😄